app/controllers/admins/employees/orders_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Admins
+  module Employees
+    class OrdersController < AdminController
```

Comment:
tu masz zly scope admin widzi wszystkie ordery, nie powinny byc w module employees

Reply:
czyli jednak orders nie mają być zagnieżdżone w employyes ? bo generalnie wysyłasz mi trochę mixed signals w tej sprawie :Oskar Lakner — Wczoraj o 14:02
potrzebujesz do tego:
1. nested route employees -> orders z akcja index (to lista wszystkich zamowien tego usera) oraz show (pojedyncze zamowienie)